JMS/README.md

```diff
@@ -225,6 +225,16 @@ If you have used maven to build the samples, you can run
 
 The response sample will get a message from the queue, process it and put the response on the reply to queue and keep looking for more messages to respond to till you ctrl+c interrupt it.
 
+### Poison Messages
+While running the request/response samples, if an error, such as "Reply to Queue no longer exists, skipping request" or "Reply to destination is invalid", is received then the message must be a Poison Message. This occurs when the message cannot get processed by the receiving application and gets sent back to the queue. The application will still keep trying to send the message and it will keep getting put back in the queue, like an endless loop.
```

Comment:
remove the sentence:

> The application will still keep trying to send the message

as this is not true. The application has already sent the message, and it is already on the queue. Other applications will keep on seeing it until it is removed.